lib/napa/deprecations/application_api.rb

```diff
@@ -1,8 +1,10 @@
 module Napa
   class Deprecations
+    LINK = 'https://github.com/bellycard/napa/blob/master/lib/napa/generators/templates/scaffold/app/apis/application_api.rb'
```

Comment:
Looks like the file got move to here: https://github.com/bellycard/napa/blob/master/lib/napa/cli/templates/new/app/apis/application_api.rb